pkg/registry/etcd/etcd_test.go

```diff
@@ -33,32 +33,35 @@ import (
 	podetcd "github.com/GoogleCloudPlatform/kubernetes/pkg/registry/pod/etcd"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/tools"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/tools/etcdtest"
 
 	"github.com/coreos/go-etcd/etcd"
 )
 
-func NewTestEtcdRegistry(client tools.EtcdClient) *Registry {
-	registry := NewRegistry(tools.NewEtcdHelper(client, latest.Codec), nil, nil)
-	return registry
+func NewTestEtcdRegistry(client tools.EtcdClient) (*Registry, tools.EtcdHelper) {
```

Comment:
does the second output parameter get used anywhere? The diff will be smaller if you don't add it.